modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/jobs/VersionMigrationsWorker.kt

```diff
@@ -119,51 +126,20 @@ class VersionMigrationsJob : JobService() {
         }
     }
 
-    @Inject lateinit var podcastManager: PodcastManager
-
-    @Inject lateinit var episodeManager: EpisodeManager
-
-    @Inject lateinit var settings: Settings
-
-    @Inject lateinit var fileStorage: FileStorage
-
-    @Inject lateinit var appDatabase: AppDatabase
-
-    @Inject lateinit var playbackManager: PlaybackManager
-
-    @Volatile private var shouldKeepRunning = true
-
-    override fun onStartJob(jobParameters: JobParameters): Boolean {
-        LogBuffer.i(LogBuffer.TAG_BACKGROUND_TASKS, "VersionMigrationsTask onStartJob")
-
-        Thread {
-            var shouldReschedule = false
-            try {
-                if (!shouldKeepRunning) {
-                    shouldReschedule = true
```

Comment:
I have removed this reschedule logic as `WorkManager` is more robust in doing work in the background. If the task fails the first time, I don't see a reason why it wouldn't fail in the next run. 🤔